app/assets/stylesheets/vat-repayment-tracker.css

```diff
@@ -296,10 +296,6 @@ details summary:focus,
     text-decoration: none;
 }
 
-.js-enabled .govuk-tabs__tab:link, .js-enabled .govuk-tabs__tab:visited, .js-enabled .govuk-tabs__tab:hover, .js-enabled .govuk-tabs__tab:active, .js-enabled .govuk-tabs__tab:focus {
-    color: #0b0c0c;
```

Comment:
Is this related to making the font colour black when the yellow focus is on it? Without it, wouldn't it default back to white font on yellow focus?